apps/desktop/electron/main/services/export-service.ts

```
@@ -1263,6 +1265,185 @@
             });
         }
 
+        // Helper: convert rich-text HTML notes into a markdown-like string for the appendix.
+        const htmlToMarkdownForAppendix = (html: string): string => {
+            if (!html) return "";
+
+            // Strip control characters that can confuse PDF text rendering.
+            let text = html.replace(/[\u0000-\u001F\u007F-\u009F]/g, "");
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)